client/packages/programs/src/JsonForms/useJsonForms.tsx

```diff
@@ -12,27 +12,21 @@ import { JsonFormsRendererRegistryEntry } from '@jsonforms/core';
 import {
   EncounterLineChart,
   encounterLineChartTester,
-} from './components/EncounterLineChart';
-import { BMI, bmiTester } from './components/BMI';
-import { DateOfBirth, dateOfBirthTester } from './components/DateOfBirth';
-import { IdGenerator, idGeneratorTester } from './components/IdGenerator';
-import {
+  BMI,
```

Comment:
Tidied up the imports by creating an index.ts file

Reply:
You have to edit two files instead of one now...

Reply:
Two tidy files instead of one shambles of unreadable imports 😉

(Plus, it's consistent with everywhere else)